scss/civicrm/common/_ui-dialog.scss

```diff
@@ -43,8 +46,12 @@
   }
 
   .ui-dialog-content#bootstrap-theme,
-  .ui-dialog-content[id*="crm-ajax-dialog"] {
+  .ui-dialog-content[id*='crm-ajax-dialog'] {
     background: $crm-white;
     padding: 10px 20px;
   }
 }
+
+.ui-widget-overlay.ui-front {
```

Comment:
This PR is about making sure that the old UI modal appear on top of *bootstrap* modals, meaning that all the CSS changes should be in the bootstrap-side of the theme (bootstrap/ folder), not in the legacy side (civicrm/ folder).

There is a [bootstrap/overrides/crm](https://github.com/compucorp/org.civicrm.shoreditch/tree/staging/scss/bootstrap/overrides/crm) folder explicitly for overriding crm styles when inside the bootstrap namespace